gradle/settings/tomcat10.gradle

```diff
@@ -47,7 +47,16 @@ buildscript {
 import org.labkey.gradle.util.BuildUtils
 
 // Add modules here as they are migrated to tomcat 10
-BuildUtils.includeModules(this.settings, rootDir, ["server/modules/platform"], [], true)
+BuildUtils.includeModules(this.settings, rootDir, [
+        "server/modules/cas",
+        "server/modules/cloudServices",
+        "server/modules/commonAssays",
+        "server/modules/dataintegration",
+        "server/modules/mfa",
+        "server/modules/platform",
+        "server/modules/premium",
+        "server/modules/saml",
```

Comment:
We can add assayRequest, ldap, and redshift as well. They required no changes, but they build and run.

Reply:
done